src/main/java/com/schematica/world/schematic/SchematicAlpha.java

```diff
@@ -101,7 +102,7 @@ else if (tagCompound.hasKey(Names.NBT.ADD_BLOCKS_SCHEMATICA))
                     pos.setPos(x, y, z);
                     try
                     {
-                        final IBlockState blockState = block.getStateFromMeta(meta);
+                        @SuppressWarnings("deprecation") final IBlockState blockState = block.getStateFromMeta(meta);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotation 'SuppressWarnings' should be alone on line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)